README.md

```diff
@@ -52,13 +54,89 @@ Master and Worker nodes:
 3. In the Launch Stack dialog, provide the required information (stack name, password, flavor size, external network of your environment, etc.).
 4. Click Launch to launch the stack. This creates all required resources for running SUSE CaaS Platform in an OpenStack environment. The stack creates one Admin Node, one Master Node, and server worker nodes as specified.
 
-### INSTALL TEMPLATES FROM THE COMMAND LINE
+### INSTALL TEMPLATES FROM THE COMMAND LINE 
 
 1. Specify the appropriate flavor and network settings in the caasp-environment.yaml file.
 2. Create a stack in Heat by passing the template, environment file, and parameters:
 ```
   $ openstack stack create -t caasp-stack.yaml -e caasp-environment.yaml \
-    --parameter image=CaaSP-2 caasp-stack
+    --parameter image=CaaSP-3 caasp-stack
+```
+
+### INSTALL TEMPLATES FROM COMMAND LINE (FOR MULTIPLE MASTER NODES)
+
+1. Specify the appropriate flavor and network settings in caasp-multi-master-environment.yaml file.
+2. Set master_count to more desired number in caasp-multi-master-environment.yaml file
+e.g.
+
+``` 
+master_count: 2
+```
+3. In a multi master deployment, load balancer is created which points to multiple master nodes for ports
+   32000 and 6443. By default the load balancer provider is 'octavia'. 
```

Comment:
I believe you mentioned some additional steps are required to enable octavia? Would be useful to have them documented.

Reply:
Ok let me add steps to enable octavia to the README